### PR TITLE
Fix issue push at last opcode

### PR DESCRIPTION
**Pull request by ignasirv**
Fixed issue where last opcode of a contract is a push following issue https://github.com/0xPolygonHermez/cdk-erigon/issues/1073

### CODE REVIEW

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Review by krlosMata — APPROVED** [review]

File: main/opcodes/stack-operations.zkasm [hunk 1]
```diff
@@ -127,16 +127,17 @@ opPUSH32:
...
     $ => B              :MLOAD(bytecodeLength)
     PC + D - B          :JMPN(opAuxPUSHA2)
-    B - PC => D
+    B - PC => D,E       :JMPZ(lastOpcodeIsPush)
 
 opAuxPUSHA2:
-    ; check out-of-gas
-    GAS - %GAS_FASTEST_STEP => GAS      :JMPN(outOfGas)
     ; read bytes from calldata
     D => E              :CALL(readPush); in: [D,E: bytes to read] out: [E: value read]
```
Comment (jesusholoo) [inline main/opcodes/stack-operations.zkasm:139]: D => E              :CALL(readPush); in: [D,E: bytes to read] out: [E: value read]

Change comment, only in D, E it's just output

**Review by jesusholoo — APPROVED** [review]

File: main/opcodes/stack-operations.zkasm (hunk 1 above)
Comment (ignasirv) [reply main/opcodes/stack-operations.zkasm:139]: Both are used at readPush as input

Comment (jesusholoo) [reply main/opcodes/stack-operations.zkasm:139]: true ^^

**Pull request merged**